cogs/source/source.py

```diff
@@ -3,39 +3,42 @@
 from tracemoepy.errors import EmptyImage, EntityTooLarge, ServerError, TooManyRequests
 
 
+def messageBuilder(titleEnglish: str, anilistID: str, episode: str, similarity: int):
+    """
+    Builds the message that is sent in response
+    """
```

Comment:
nit: could we put this docstring in one line?
```suggestion
def messageBuilder(titleEnglish: str, anilistID: str, episode: str, similarity: int):
    """Builds the message that is sent in response"""
```